app/assets/javascripts/host_edit.js

```diff
@@ -547,12 +547,17 @@ $(document).on('change', '.interface_domain', function () {
 });
 
 $(document).on('click', '.suggest_new_ip', function (e) {
-  clearError($(this).closest('fieldset').find('.interface_ip'));
+  var ip_field = $(this).closest('fieldset').find('.interface_ip');
+  clearError(ip_field);
+  ip_field.val('');
   interface_subnet_selected($(this).closest('fieldset').find('select.interface_subnet'));
   e.preventDefault();
 });
 
 $(document).on('change', '.interface_subnet', function () {
+  var ip_field = $(this).closest('fieldset').find('.interface_ip');
```

Comment:
Since this code is the same as the change above, how about extracting it to a function `clearIPField`?